tests/virtualization/virtman_view.pm

```diff
@@ -45,6 +45,9 @@ sub run {
     send_key 'alt-c';
     wait_screen_change { send_key 'esc' };
 
+    # Make sure we have virt-manager window
+    assert_screen 'virt-manager';
```

Comment:
If you want to introduce the `assert_screen` then please delete the `wait_screen_change` just above as it is not needed anymore and a time waste

Reply:
Ok, I will try that.